static/js/unread_ui.js

```diff
@@ -53,9 +53,14 @@ exports.update_unread_counts = function () {
     top_left_corner.update_dom_with_unread_counts(res);
     stream_list.update_dom_with_unread_counts(res);
     pm_list.update_dom_with_unread_counts(res);
-    notifications.update_title_count(res.home_unread_messages);
     notifications.update_pm_count(res.private_message_count);
 
+    if (page_params.desktop_icon_count_display === 2) { // DESKTOP_ICON_COUNT_DISPLAY_NOTIFIABLE
```

Comment:
Same comment on the magic number 2 here.

Reply:
Fixed, thanks!